mmv1/templates/terraform/examples/tpu_v2_queued_resource_basic.tf.tmpl

```diff
@@ -0,0 +1,16 @@
+resource "google_tpu_v2_queued_resource" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+
+  name = "{{index $.Vars "qr_name"}}"
+  zone = "us-central1-c"
+
+  tpu {
+    node_spec {
+      parent = "projects/{{index $.TestEnvVars "project"}}/locations/us-central1-c"
+      node_id = "{{index $.Vars "tpu_name"}}"
+      node {
+        runtime_version = "tpu-vm-tf-2.13.0"
```

Comment:
Could you please update this example to include the fields tpu.node_spec.node.description & tpu.node_spec.node.accelerator_type, [mentioned here](https://github.com/GoogleCloudPlatform/magic-modules/pull/12277#issuecomment-2463432967)?

Reply:
Done, updated the basic example to include filled out fields tpu.node_spec.node.description and tpu.node_spec.node.accelerator_type